### PR TITLE
Show key usage and extended key usage

### LINKED ISSUE #10: Show X.509v3 key usage and extended key usage
Should show X.509v3 key usage and extended key usage (if present) for a certificate.

### PR DESCRIPTION
**Pull request by csstaub**
r: @christodenny @alokmenghrajani @mcpherrinm 
Show key usage and extended key usage. Fixes #10.

### CODE REVIEW

File: display.go [hunk 1]
```diff
@@ -107,6 +142,26 @@ func highlightAlgorithm(sig x509.SignatureAlgorithm) string {
 	return color.SprintFunc()(sig.String())
 }
 
+// keyUsage decodes/prints key usage from a certificate.
+func keyUsage(ku x509.KeyUsage) []string {
+	out := []string{}
+	for key, value := range keyUsageStrings {
+		if ku&key > 0 {
```
Comment (christodenny) [inline display.go]: Could the bitwise and be separated out? Better readability?

Comment (christodenny) [inline display.go]: Other than that lgtm

Comment (csstaub) [inline display.go]: This is how `gofmt` formats it. It's the one true way to space things.

Comment (christodenny) [inline display.go]: but readability :'(

**Pull request merged**